integrations/gcp/gcp_core/search/iterators.py

```diff
@@ -0,0 +1,17 @@
+from typing import Any, AsyncIterable, Callable
+from port_ocean.core.ocean_types import ASYNC_GENERATOR_RESYNC_TYPE
+from port_ocean.utils.async_iterators import stream_async_iterators_tasks
+from gcp_core.search.searches import search_all_projects
+
+
+async def iterate_per_available_project(
```

Comment:
this can move to the utils

Reply:
I disagree. in future versions we might want to add other iterators to collect more data.

Reply:
dont think 2 far ahead when we will get to this date we will split the utils file